code-injection-analysis/ajinabraham_Node.Js-Security-Course/eval.js

```
@@ -0,0 +1,7 @@
+var express = require('express');
+var app = express();
+app.get('/', function(req, res) {
+  var resp=eval("("+req.query.name+")");
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)